core/observe.c

```diff
@@ -1493,4 +1493,33 @@ bool observe_handleNotify(lwm2m_context_t * contextP,
     }
     return true;
 }
+
+#ifndef LWM2M_VERSION_1_0
```

Comment:
It might be useful to add a check for `LWM2M_SERVER_MODE` in addition to `LWM2M_VERSION_1_0`.

Reply:
I made the change